README.md

```diff
@@ -1,91 +1,130 @@
-# Determined: Deep Learning Training Platform
-
 <p align="center"><img src="determined-logo.png" alt="Determined AI Logo"></p>
 
-Determined is an open-source deep learning training platform that makes building
-models fast and easy. Determined enables you to:
+Determined is an all-in-one deep learning platform, compatible with PyTorch and TensorFlow.
 
-* **Train models faster** using state-of-the-art distributed training, without
-  changing your model code
-* **Automatically find high-quality models** with advanced hyperparameter tuning
-  from the creators of Hyperband
-* **Get more from your GPUs** with smart scheduling and cut cloud GPU costs by
-  seamlessly using preemptible instances
-* **Track and reproduce your work** with experiment tracking that works
-  out-of-the-box, covering code versions, metrics, checkpoints, and
-  hyperparameters
+It takes care of:
 
-Determined integrates these features into an easy-to-use, high-performance deep
-learning environment — which means you can spend your time building models
-instead of managing infrastructure.
+- **Distributed training**, for faster results.
+- **Hyperparameter tuning**, for obtaining the best models.
+- **Resource management**, for cutting cloud GPU costs.
+- **Experiment tracking**, for analysis and reproducibility.
```

Comment:
- Distributed training for faster results.
- Hyperparameter tuning for obtaining the best models.
- Resource management for cutting cloud GPU costs.
- Experiment tracking for analysis and reproducibility.